contracts/V1/StateChannelDiamondProxy/StateSnapshotFacet.sol

```diff
@@ -65,8 +66,10 @@ contract StateSnapshotFacet is StateChannelCommon {
             _isDisputeCommitmentValid(dispute, disputeTimestamp, channelId),
             ErrorDisputeProofNotValid()
         );
+        // TODO: add addressesInThreshold
+        // addressesInThreshold are? snapshot.participents - onchainSlasshes?
```

Comment:
@lukastanisic99 
notice code is incomplete here, since i am not sure where to get the participants for early finalization from
- the current on-chain snapshot?
- genesis state snapshot of the dispute? (needs to be oracelized as well)

Reply:
Hopefully my previous comment above also answers this